acmw-db-app/pages/signin.js

```diff
@@ -0,0 +1,129 @@
+import Head from 'next/head'
+import styles from '../styles/SignIn.module.css'
+import React from 'react';
+
+function SignIn() {
+  // TODO link to page that sends password by email?
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>ACM-W Database Sign In</title>
+      </Head>
+
+      <main className={styles.main}>
+        <div className={styles.card}>
+          <img src="/logo.png" alt="ACM-W Logo" className={styles.logo} />
+          <h1> OSU Database </h1>
+          <h2> Please sign in to continue </h2>
+          <TextInput
+            id="username"
+            label="Username"
+            predicted="Brutus Buckeye"
+          />
+          <div className={styles.passwordChunk}>
+            <TextInput
+              id="password"
+              label="Password"
+              type="password"
+            />
+            <a className={styles.smol} href="https://crouton.net/">Forgot username or password?</a>
+          </div>
+          <SignInButton
+          />
+        </div>
+      </main>
+    </div>
+  )
+}
+
+export default SignIn
+
+class TextInput extends React.Component {
+
```

Comment:
I'm wondering if this could be in its own component file (should we have a components folder?) since it seems like it could be reusable. Maybe not though if you have appearance logic (could be renamed to be specific to SignInTextInput or something?)

Reply:
Yeah I was thinking about that too. Maybe we can make a more general one and then extend that one here?